tests/integration/synapseclient/integration_test.py

```diff
@@ -86,7 +86,6 @@ async def testCustomConfigFile(schedule_for_cleanup):
         )
 
 
-@pytest.mark.flaky(reruns=3, only_rerun=["SynapseHTTPError"])
```

Comment:
Interesting, how have these been resolved?

Reply:
It has not been explicitly, but a global retry mechanism was put in place after this was written.